introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello, my name is Heather. I'm here to chew bubblegum and master Github. And I'm all out of bubblegum.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?